tests/multicast.sh

```diff
@@ -28,7 +28,7 @@ for INTERFACE in $(echo $INTERFACES | xargs); do
     fi
 done
 
-LOCATORS="udp/224.0.0.225:7447#iface=$INTERFACE"
+LOCATORS="udp/224.0.0.225:7446#iface=$INTERFACE"
```

Comment:
This one should not be changed.
These are the tests for the multicast transport, so it should run on 7447 as defined for the unicast transport.